### PR TITLE
Added Movie List database

### PR DESCRIPTION
**Pull request by ADITIRAI290698**
- [Docs](https://github.com/FifthTry/ftd.dev/blob/main/h1/index.ftd)

Added 1 database wrt to response, movie and record.

### CODE REVIEW

File: dj/movies/views.py [hunk 1]
```diff
@@ -4,9 +4,15 @@
...
-    return django.http.JsonResponse(d, status=200)
+    record_response = {"page-number":1,"next":"next", "previous":"previous", "count":5,"list movies":"bj"}
 
+    record_movie = {"id": 1, "title": "Laal Singh Chaddha","release-date": "11 August 2022", "poster": "Laal Singh Chaddha Movies",
+         "director": "Advait Chandan", "description": "LAAL SINGH CHADDHA, a simple man whose extraordinary journey will fill you with love, warmth & happiness.", "rating" : 6, "review-count": 2, "list reviews": "bj"  }
+
+
+    record_review = {"id": 1, "title": "Laal Singh Chaddha", "description": "Very cheap version of the Original movie. Very slow screenplay.Unnecessary things are forced to the movie. Lack of logic, lack of emotions.","reviewer": "imDb", "rating": 5}
+
+    return django.http.JsonResponse([record_response,record_movie,record_review], status=200)
```
Comment (amitu) [inline dj/movies/views.py]: Use `black` to format source code. https://pypi.org/project/black/

File: dj/movies/serializers.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
```
Comment (AbrarNitk) [inline dj/movies/serializers.py:1]: Remove `rest_framework` we don't require this. I have already sent you a template please use that. `django.http.JsonResponse` will work.

File: dj/movies/views.py [hunk 3]
```diff
@@ -1,11 +1,107 @@
 import django.http
+from .models import Movie
+from .serializers import MovieSerializer
```
Comment (AbrarNitk) [inline dj/movies/views.py:3]: Remove serializers

File: dj/movies/views.py [hunk 4]
```diff
@@ -1,11 +1,107 @@
...
 
 # Create your views here.
 
 
-def movie_list(req: django.http.HttpRequest):
-    d = {"name": "Aditi Rai"}
-    return django.http.JsonResponse(d, status=200)
+
+
+@api_view(['GET','POST'])
```
Comment (AbrarNitk) [inline dj/movies/views.py:15]: Remove this `api_view`

**Review by AbrarNitk — CHANGES_REQUESTED** [review]

**Comment by AviralVerma13** [comment]
This PR is redundant and not is use now. refer https://github.com/FifthTry/movie-db/pull/7 now

**Pull request closed without merging**